CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Removed Features
 - ROX-18840: Sunburst widgets in the Compliance section have been removed (deprecation announced in version 4.2 release notes)
 
-### Deprecated Fatures
+### Deprecated Features
```

Comment:
Shouldn't this be in removed? Since it was already deprecated in 4.2

Reply:
Oh whoops, it needs to be above in removed